test-suite/README.md

```diff
@@ -0,0 +1,17 @@
+# Coq Test Suite
+
+The test suite can be run from the root directory by `make test-suite`.
+
+From this directory, `make aaa/bbb/ccc.v.log` runs one test (if not already run), storing the output in the named `.log` file.
```

Comment:
Might be useful to mention the `-B` flag of make, which I believe tells it to remake the target even if it's already there.

Reply:
I wasn't familiar with that option. Added.